be/src/common/config.h

```diff
@@ -763,6 +763,9 @@ CONF_Int32(quick_compaction_batch_size, "10");
 // do compaction min rowsets
 CONF_Int32(quick_compaction_min_rowsets, "10");
 
+// Avro schema file path, set default to "${DORIS_HOME}/conf/avro_schema.json"
+CONF_String(avro_schema_file_path, "${DORIS_HOME}/conf/avro_schema.json");
```

Comment:
So that each Doris cluster only support one avro schema?

Reply:
should have a property of routine load named `avro_schema_name`, and the schema should be put on every BE.

Reply:
> The old query engine with `tuple` and `RowBatch` interface will be deprecated in the future. So you'd better implement this feature in vec engine, with `block` interface.

Hi, support Vectorized `VAvroScanner` now.